api_tests/actions/views/test_action_list.py

```diff
@@ -133,6 +151,17 @@ def test_create_permissions(
         assert preprint.machine_state == 'accepted'
         assert preprint.is_published
 
+    def test_preprint_with_draft_node(self, app, url, preprint_without_node, moderator):
```

Comment:
```suggestion
    def test_preprint_without_node(self, app, url, preprint_without_node, moderator):
```
(just to avoid lies)